features/datum.feature

```diff
@@ -32,59 +32,60 @@ Functionaliteit: leveren van een datum
       | burgerservicenummer | 555550001                          |
       | fields              | burgerservicenummer,<groep>.<veld> |
       Dan heeft de persoon met burgerservicenummer '555550001' de volgende '<groep>' gegevens
-      | naam            | waarde     |
-      | <veld>.type     | <type>     |
-      | <veld>.datum    | <datum>    |
-      | <veld>.jaar     | <jaar>     |
-      | <veld>.maand    | <maand>    |
-      | <veld>.onbekend | <onbekend> |
+      | naam               | waarde       |
+      | <veld>.type        | <type>       |
+      | <veld>.datum       | <datum>      |
+      | <veld>.jaar        | <jaar>       |
+      | <veld>.maand       | <maand>      |
+      | <veld>.onbekend    | <onbekend>   |
+      | <veld>.langFormaat | <langFormaat |
```

Comment:
`<langFormaat` moet zijn `<langFormaat>`

Reply:
Aangepast

Reply:
@JohanBoer  je had hier gezegd "Aangepast", maar het is niet aangepast. Nu is deze gemerged. Kan je dit nog op de een of andere manier corrigeren?

Reply:
> @JohanBoer je had hier gezegd "Aangepast", maar het is niet aangepast. Nu is deze gemerged. Kan je dit nog op de een of andere manier corrigeren?

Ik heb het al gecorrigeerd in #1046. Ik ga de aanpassingen zo committen